lib/mysql.js

```diff
@@ -641,6 +642,24 @@ MySQL.prototype.count = function count(model, callback, where) {
 
 };
 
+MySQL.prototype.update =
+MySQL.prototype.updateAll = function(model, where, data, cb) {
+  var table = this.tableEscaped(model);
+  var fields = this.toFields(model, data);
+  var whereClause = this.buildWhere(model, where);
+  var query = util.format('UPDATE %s SET %s %s', table, fields, whereClause);
+  this.query(query, function(err, info) {
+    // @superkhau The second arg is purposely set to an empty array for
+    // consistency across all connectors. The postgres connector currently
+    // returns `data.rows` (which is an array) in the second arg and cannot be
+    // changed for backwards compatibility reasons. So instead, we leave the
+    // second arg alone and pass metadata in the third arg instead.
+    cb && cb(err, [], {
+      count: info.affectedRows
+    });
```

Comment:
See my other comment, we need to preserve backwards compatibility of the mysql connector too. How was this implemented before? Was the function provided by juggler or was `updateAll` simply not working at all?